.github/workflows/backport.yml

```diff
@@ -17,6 +17,6 @@ jobs:
         run: |
           backport-assistant backport
         env:
-          BACKPORT_LABEL_REGEXP: "backport/(?P<target>\\d+\\.\\d+\\.\\w+)"
+          BACKPORT_LABEL_REGEXP: "backport/(?P<target>\\d+\\.\\d+\\.[+\\w]+)"
```

Comment:
So the goal is to match things like `backport/1.3.+abc`?  Seems a bit weird to have a `+` right after `.`.

Reply:
I think we chatted on Slack but to summarize here, that would be weird but I'm not too worried given the limited usecase to release branches, and having a limited number of release tags.